src/library/basesqltablemodel.cpp

```diff
@@ -723,6 +725,17 @@ QVariant BaseSqlTableModel::data(const QModelIndex& index, int role) const {
     // Format the value based on whether we are in a tooltip, display, or edit
     // role
     switch (role) {
+        case Qt::BackgroundRole:
+            if (column != fieldIndex(ColumnCache::COLUMN_LIBRARYTABLE_COLOR)) {
```

Comment:
This condition unnecessarily complicates the code. It evaluates to false for all but a single column. Invoking index.siblingAtColumn() unconditionally is preferred.

Reply:
I need to come up with another solution anyway, looks like `siblingAtColumn` is not supported by our CI builds.

Reply:
I noticed that we use `index.sibling(index.row(), column)` as a workaround, only a minor inconvenience.